style1.css

```diff
@@ -0,0 +1,462 @@
+}
+
+.bar{
+    color: rgb(129, 128, 128);
+    font-weight: bold;
+}
+
+.upperRightImages{
+    text-align: end;
+    margin-top: 5px;
+}
+
+.upperRightImages:hover{
+    cursor: pointer;
+}
+
+.fbLogo{
+    width: 81px;
+    height: 20px;
+}
+.twitterLogo{
+    width:61px;
+    height:20px;
+}
+
+
+.ul{
+    display: flex;
+}
+
+.navLinks{
+    list-style: none;
+    padding-left:4px;
+}
+
+.nav{
+    display: flex;
+    justify-content: space-between;
+    margin: 0.8rem 0.5rem 0.28rem 0.5rem;
+}
+
+.menuLeft{
+    background-color:  rgb(26, 196, 248);
+    color: white;
+    text-decoration: none;
+    padding: 5px 12px 4px 12px;
+    margin: -1.3px;   
+}
+.menuLeft:hover{
+    background-color:  rgb(17, 169, 216); 
+    padding-bottom: 6px;
+}
+
+.menuRight{
+    background-color:  rgb(94, 190, 56);
+    color: white;
+    text-decoration: none;
+    padding: 5px 12px 4px 12px;
+    margin: -1.3px;
+}
+.menuRight:hover{
+    background-color: rgb(70, 177, 28);
+    padding-bottom: 6px;
+}
+
+.senior{
+    position: relative;   
+}
+
+.dropDown{
+    border: 6px solid rgb(17, 169, 216);
+    width: 250px;
+    margin-top: 6px;
+    position: absolute;
+    top: 100%;
+    left: 0;
+    background-color: #fff;
+    text-transform: capitalize;
+    display: none; 
+}
+
+
+.ddInner{
+    margin: 12px 0 20px 3px;
+}
+
+.ddlist{
+    list-style: none;
+    padding: 2px 0 2px 15px;
+    display: block;
+    font-size: 15px;
+}
+
+.ddAtag{
+    text-decoration: none;
+    color: rgb(32, 32, 32);
+}
+
+.ddBold{
+    font-size: 17px;
+    font-weight: 600;
+    color: rgb(68, 67, 67);
+}
+
+.ddflex{
+    width: 210px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.womenImg{
+    height: 50px;
+    margin: -1.8rem 0 1px 0;
+
+}
+
+.ddHr{
+    align-items: center;
+    width: 220px;
+    border:0.5px dashed rgb(121, 194, 216);
+    margin: 5px 0 5px 5px;
+}
+
+.radiusLeft{
+    border-radius: 5px 0px 0px 0px;
+}
+
+.radiusRight{
+    border-radius: 0px 5px 0px 0px;
+}
+
+.divider{
+    margin-top: 6px;
+    border: 4px solid rgb(70, 177, 28);
+}
+
+.searchBar{
+    display: flex;
+    background-color: rgb(217, 225, 240);
+    padding: 12px 20px;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.leftBar{
+    background: none;
+    
+}
+
+.lightGreyText{
+    color: rgb(59, 59, 59);
+    font-weight: 700;
+    font-size: 16.5px;  
+    margin-right: 5px;
+}
+
+.leftButtonBar{
+    padding: 2px 10px 3px 10px;
+    color: white;
+    font-size: 14.5px;
+    margin-left: 7px;
+    background-color: rgb(230, 105, 55);
+    border: 3px solid white;
+}
+
+.rightBar{
+    margin-left: 4rem;
+    background-color: rgb(217, 225, 240);
+    padding-left: 5px;
+}
+
+.selector{
+    margin-right: 1rem;
+}
+
+.zip{
+    width: 80px;
+}
+
+.searchBtn{
+    margin-left: 7px;
+    padding: 2px 15px 3px 15px;
+    background-color: rgb(94, 190, 56);
+    border: 2px solid white;
+    color: white;
+    font-size: 14.5px;
+}
+
```

Comment:
why so unwanted spaces ? Remove it

Reply:
Sorry sir, I forgot to delete that
I'm correct it right now